### PR TITLE
Py312 qt6

### PR DESCRIPTION
**Pull request by looooo**
only a test to see where we are. Locally there were qt-main and qt6-main installed into the host-env. No idea where this is coming from.

We need also to wait for:
- qt6-webengine: https://github.com/conda-forge/qt-webengine-feedstock/issues/42 *
- pivy (with qt6): https://github.com/conda-forge/pivy-feedstock/pull/44

@wwmayer is it possible to build freecad without qtwebengine?

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: .ci_support/linux_64_python3.12.____cpython.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    max_pin: x.x
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.12.* *_cpython
+qt6_main:
+- '6.6'
+qt_main:
+- '5.15'
```
Comment (adrianinsaval) [inline .ci_support/linux_64_python3.12.____cpython.yaml:34]: could this be why both get installed?

**Comment by adrianinsaval** [comment]
yaml-cpp is also required for 0.22

File: .ci_support/linux_64_python3.12.____cpython.yaml (hunk 1 above)
Comment (looooo) [reply .ci_support/linux_64_python3.12.____cpython.yaml:34]: this wasn't a problem for pivy, so I guess this should not be the problem. Maybe one of the dependencies of freecad pulls qt-main.

Comment (looooo) [reply .ci_support/linux_64_python3.12.____cpython.yaml:34]: @conda-forge/core any ideas why qt-main and qt6-main are installed for the host-env?

Comment (adrianinsaval) [reply .ci_support/linux_64_python3.12.____cpython.yaml:34]: I think pivy pulls qt-main, we need pivy compiled with pyside6 for python 12

Comment (looooo) [reply .ci_support/linux_64_python3.12.____cpython.yaml:34]: I did the same with pivy here: https://github.com/conda-forge/pivy-feedstock/pull/44

Comment (looooo) [reply .ci_support/linux_64_python3.12.____cpython.yaml:34]: btw.: @adrianinsaval can you unpin coin in the freecad-bundle. so we get some users test if everything is ok with coin4.0.2 which should be available soon. I moved the problematic coin4.0.1 win build to broken, so it shouldn't be pulled anyways.

Comment (looooo) [reply .ci_support/linux_64_python3.12.____cpython.yaml:34]: thanks @adrianinsaval  coin3d 4.0.2 is uploaded.

**Comment by wwmayer** [comment]
> is it possible to build freecad without qtwebengine?

Yes, but you have to switch off the Web and Start modules.

File: .ci_support/linux_64_python3.12.____cpython.yaml (hunk 1 above)
Comment (looooo) [reply .ci_support/linux_64_python3.12.____cpython.yaml]: Good idea. We should use vtk-base in general

**Comment by looooo** [comment]
https://github.com/conda-forge/libnetcdf-feedstock/pull/191

**Comment by adrianinsaval** [comment]
@conda-forge-admin, please restart ci

**Comment by looooo** [comment]
@conda-forge-admin, please rerender

**Comment by looooo** [comment]
@adrianinsaval seems like vtk-base is also dependent on qt:
`vtk-base:                  9.2.6-qt_py312h1234567_219`

**Comment by looooo** [comment]
@conda-forge/core I think I saw that there is a possibility to specify packages which should not be installed. For this feedstock this would help a lot, as qt-main and qt6-main are pulled, but this recipe should only depend on one of them. What is the syntax to exclude a package?

**Comment by hmaarrfk** [comment]
Let's get vtk pushed to qt6 first

**Comment by looooo** [comment]
#122

**Pull request closed without merging**